src/main/java/de/kittybot/kittybot/cache/DashboardSessionCache.java

```diff
@@ -9,56 +9,38 @@
 public class DashboardSessionCache{
 
 	private static final Map<String, DashboardSession> SESSION_CACHE = new HashMap<>();
-	private static final Map<String, Boolean> USER_SESSION_CACHE = new HashMap<>();
 
 	private DashboardSessionCache(){}
 
 	public static void addSession(final DashboardSession session){
-		SESSION_CACHE.put(session.getSessionKey(), session);
-		USER_SESSION_CACHE.put(session.getUserId(), true);
+		SESSION_CACHE.put(session.getUserId(), session);
 		Database.addSession(session);
 	}
 
-	public static void deleteSession(final String sessionKey){
-		var session = SESSION_CACHE.get(sessionKey);
-		Database.deleteSession(sessionKey);
+	public static void deleteSession(final String userId){
+		var session = SESSION_CACHE.get(userId);
 		if(session == null){
 			return;
 		}
-		var userId = session.getUserId();
-		SESSION_CACHE.remove(sessionKey);
-		if(Database.getUserSessions(userId) > 1){
-			return;
-		}
+		Database.deleteSession(userId);
+		SESSION_CACHE.remove(userId);
 		GuildCache.uncacheUser(userId);
-		USER_SESSION_CACHE.remove(userId);
-	}
-
-	public static boolean sessionExists(final String sessionKey){
-		return SESSION_CACHE.containsKey(sessionKey) || getSession(sessionKey) != null;
 	}
 
-	public static DashboardSession getSession(final String sessionKey){
-		var session = SESSION_CACHE.get(sessionKey);
+	public static DashboardSession getSession(final String userId){
+		var session = SESSION_CACHE.get(userId);
 		if(session != null){
 			return session;
 		}
-		session = Database.getSession(sessionKey);
+		session = Database.getSession(userId);
 		if(session != null){
-			SESSION_CACHE.put(sessionKey, session);
+			SESSION_CACHE.put(userId, session);
 		}
 		return session;
-
 	}
 
 	public static boolean hasSession(final String userId){
-		var hasSession = USER_SESSION_CACHE.get(userId);
-		if(hasSession != null){
-			return hasSession;
-		}
-		hasSession = Database.hasSession(userId);
-		USER_SESSION_CACHE.put(userId, hasSession);
-		return hasSession;
+		return SESSION_CACHE.containsKey(userId) || Database.hasSession(userId);
```

Comment:
see, that's why the `USER_SESSION_CACHE` was a thing. `hasSession` is used by our cache policy, so it will make a db request each time jda wants to cache a member and the `userId` is not present as a key in the `SESSION_CACHE`.

Reply:
hm I see
I think doing that db request would be okay tbh 
but we could also revert that if u like

Reply:
yes, reverting removal of that cache would be the best option